### ISSUE TITLE
Cannot format due to syntax errors" not useful enough to debug

### ISSUE THREAD

**Issue by nipunn1313** [saviorisdead/RustyCode#165]
I am getting the warning "Cannot format due to syntax errors"
It would be really nice if the error message included the command which was run so I could try to repro the errors. In my case, the program compiles, so it's unclear what's going on.

From reading the code, it looks like it is calling
rustfmt --skip-children --write-mode=diff <filename> which is returning some non-3 status code. It would be great if we could see the status code and the exact command run to help repro/fix on the command line.

Thanks

**Comment by vaags** [comment]
Yes. This message is potentially misleading. When I run rustfmt manually on the same code it works just fine, so the syntax must be correct?

**Comment by vaags** [comment]
I have the same problem (VS Code 1.3 on Windows). No idea how to fix this. Running rustfmt manually works fine.

**Comment by CryZe** [comment]
This is because you guys are using rustfmt 0.6.0 which isn't yet supported by RustyCode. There's a PR for it here: https://github.com/saviorisdead/RustyCode/pull/161
